lib/js/styles/ItemsList.vue

```diff
@@ -125,17 +90,20 @@ export default {
 			TOKENS_TYPES: Object.freeze(TOKENS_TYPES),
 		};
 	},
-	mounted() {
-		this.itemsLocal.forEach((list, index) => {
-			if (list.disabled) {
-				for (let key in list.list) {
-					if (key === list.disabled) {
-						this.itemsLocal[index].list[key] = null;
-						delete this.itemsLocal[index].list[key];
-					}
-				}
-			}
-		});
-	},
+  computed: {
+    getItemsLocal() {
+      this.itemsLocal.forEach((list, index) => {
```

Comment:
I though we'll use `itemsLists` prop in the computed - create a local copy and then perform modifications.

Right now, if you change prop value, component won't be updated

Reply:
Fixed.